designsafe/static/scripts/data-depot/components/projects/publication-preview/publication-preview.scss

```diff
@@ -3,7 +3,7 @@
     .preview-tree-btn-wrapper {
         position: absolute;
         left: -175px;
-        border-bottom: 3px solid purple;
+        border-bottom: 3px solid gray;
```

Comment:
I think this was included in one of your other PRs

Reply:
This is supposed to be changed to gray. Is this doing something else?